plugins/SitesManager/SitesManager.php

```diff
@@ -538,4 +525,27 @@ public function noDataPageGTMTabInstructions(&$out)
         $view->jsTag = $jsTag;
         $out = $view->render();
     }
+
+    public function noDataPageWordpressTabInstructions(&$out)
+    {
+        Piwik::checkUserHasSomeViewAccess();
+        $view = new View("@SitesManager/_wordpressTabInstructions");
+        $faqLink = 'https://matomo.org/faq/general/faq_114/';
+        $authLink = '';
+        if (Piwik::isUserHasSomeViewAccess()) {
+            $idSite = Common::getRequestVar('idSite', 0, 'int');
```

Comment:
Just a general note that for changes going into 5.x-dev that won't be cherry picked for 4.x-dev, it would be good to use the new `core/Request` class `getStringParameter()` / `getIntegerParameter()` methods rather than Common::getRequestVar() which will be deprecated at some point.

Reply:
@bx80 Updated as suggested, can you check if that's the correct way ?

Reply:
Looks good to me :+1: